fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/meeting/CreateMeeting.java

```diff
@@ -29,9 +32,14 @@ public class CreateMeeting extends Data {
    * @throws IllegalArgumentException if the id is invalid
    */
   public CreateMeeting(
-      String laoId, String id, String name, long creation, String location, long start, long end) {
-    if (!id.equals(
-        Hash.hash(EventType.MEETING.getSuffix(), laoId, Long.toString(creation), name))) {
+      String laoId,
+      String id,
+      String name,
+      long creation,
+      @Nullable String location,
```

Comment:
Why is location nullable here if in MeetingBuilder it cannot be null?

Reply:
Essentially it's because the location is not required for the json schema of the createMeeting, so there it can be null. However, the MeetingBuilder doesn't build the createMeeting but it builds the Meeting object from the createMeeting. To be coherent with what happens also with the RollCall, to field in the Meeting object is alllowed to be null, thus the builder sets the location to empty if null in the CreateMeeting. You can observe the same behaviour with the description field for the RollCalls